AISKU/src/Initialization.ts

```diff
@@ -411,3 +427,49 @@ export class Initialization implements IApplicationInsights {
         _this.config.diagnosticLogInterval && _this.config.diagnosticLogInterval > 0 ? _this.config.diagnosticLogInterval : 10000;
     }
 }
+
+// tslint:disable-next-line
+(function () {
+    let sdkSrc = null;
+    let isModule = false;
+
+    try {
+        // Try and determine whether the sdk is being loaded from the CDN
+        // currentScript is only valid during initial processing
+        let scrpt = (document ||{} as any).currentScript;
+        if (scrpt) {
+            sdkSrc = scrpt.src;
+        // } else {
+        //     // We need to update to at least typescript 2.9 for this to work :-(
+        //     // Leaving as a stub for now so after we upgrade this breadcrumb is available
+        //     let meta = import.meta;
+        //     sdkSrc = (meta || {}).url;
+        //     isModule = true;
+        }
+    } catch (e) {
+    }
+
+    if (sdkSrc) {
+        try {
+            let url = sdkSrc.toLowerCase();
+            if (url) {
+                let src = "";
+                if (url.indexOf("://az416426.vo.msecnd.net/") !== -1) {
```

Comment:
Only populate if this is from the standard CDN -- ignore non MS defined endpoints, and only include an anonymous name "cdn1" rather than the full domain/url

Reply:
If the script is consumed from CDN via the CName will we have to update this to match those domain names?

Reply:
Yes, once we have enabled additional common names this will need to be updated and I'm planning on enumerating the known names (which is why I called the existing one cdn1)